mollie/organizations.go

```diff
@@ -0,0 +1,74 @@
+package mollie
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// Organization describes a organization detail
+type Organization struct {
+	Resource           string             `json:"resource,omitempty"`
+	ID                 string             `json:"id,omitempty"`
+	Name               string             `json:"name,omitempty"`
+	Email              string             `json:"email,omitempty"`
+	Locale             string             `json:"locale,omitempty"`
+	Address            *Address           `json:"address,omitempty"`
+	RegistrationNumber string             `json:"registrationNumber,omitempty"`
+	VatNumber          string             `json:"vatNumber,omitempty"`
+	VatRegulation      string             `json:"vatRegulation,omitempty"`
+	Links              *OrganizationLinks `json:"_links,omitempty"`
+}
+
+// OrganizationLinks describes all the possible links to be returned with
+// a organization object.
+type OrganizationLinks struct {
+	Self          URL `json:"self,omitempty"`
+	Chargebacks   URL `json:"chargebacks,omitempty"`
+	Customers     URL `json:"customers,omitempty"`
+	Invoices      URL `json:"invoices,omitempty"`
+	Payments      URL `json:"payments,omitempty"`
+	Profiles      URL `json:"profiles,omitempty"`
+	Refunds       URL `json:"refunds,omitempty"`
+	Settlements   URL `json:"settlements,omitempty"`
+	Documentation URL `json:"documentation,omitempty"`
+}
+
+// OrganizationsService instance operates over organization resources
+type OrganizationsService service
+
+// Get retrieve an organization by its id.
+func (is *OrganizationsService) Get(id string) (i Organization, err error) {
```

Comment:
You should return a pointer to the organization.

Also `is` is not descriptive for organization service I will rather use `os` or something else, IDK if os will be conflictive naming with the standard library `os` package.
```suggestion
func (os *OrganizationsService) Get(id string) (i *Organization, err error) {
```